bagel/utilities/file_utils.py

```diff
@@ -94,3 +95,24 @@ def save_jsonld(data: dict, filename: Path):
     with open(filename, "w") as f:
         f.write(json.dumps(data, indent=2))
     logger.info(f"Saved output to:  {filename}")
+
+
+def request_file(url: str, backup_path: Path) -> tuple[list, str | None]:
+    contents = []
+    err = None
+
+    try:
+        response = httpx.get(url)
+        response.raise_for_status()
+        contents = response.json()
+    except (httpx.HTTPError, json.JSONDecodeError) as request_err:
+        try:
+            # We don't use file_utils.load_json() here because we don't want to throw an error yet if there are problems with the file.
+            # Otherwise, there may be exceptions on import, meaning there will be errors even if the user runs just bagel --help.
```

Comment:
This is great, but you should add that `request_file` is called on import because it is a top-level function call in `mappings.py`. Or at least some kind of pointer as to why this blameless function should have such serious consequences